spk/debian-chroot/Makefile

```diff
@@ -26,28 +26,37 @@ INSTALL_PREFIX = /usr/local/$(SPK_NAME)
 
 POST_STRIP_TARGET = debian-chroot_extra_install
 
+
+include ../../mk/spksrc.spk.mk
+
+ifeq ($(findstring $(ARCH),88f5281 powerpc ppc824x ppc853x ppc854x dockerx64 grantley kvm64),$(ARCH))
```

Comment:
I am a bit surprised these specific `x64` archs don't work?

@ymartin59 Seems to me all reference to `88f5281` should be removed from the repo, since it doesn't even have a toolchain anymore.

Reply:
just reverted the chroot updates to get rid of this dependency.
The real problem is that ARCH is not defined in this context.

Reply:
Huh? `ARCH` is not defined due to the `include` moving up?

Reply:
No, it has nothing to do with the include. `ARCH` is even not defined with the source of the current master.
The only difference to working packages is, that the `Makefile` located below /spk has no dependency to any cross package.
Perhaps the arch-%: target with indirect call of make is the key (see spksrc.cross-cc.mk line 121, spksrc.spk.mk line 420) as far as I understand the `MAKEFLAGS` construct.

Reply:
Yes the `ARCH` only gets defined when you run any of the `make arch-xxx-6.1` or any of the other commands. Then it will deduct the  `ARCH` from it. 
So when we get to package `Makefile` it will be defined and you can use it, just like you were doing.
But still not sure why those specific `x64` arch's needed to be excluded?